configs/azure.json

```diff
@@ -3,8 +3,8 @@
     "description": "Azure estimates based on April 2019 advertised prices",
     "CPU": "0.03900",
     "spotCPU": "0.007764",
-    "RAM": "0.001917",
-    "GPU": "0.0428925",
+    "RAM": "0.00400",
+    "GPU": "0.01828925",
```

Comment:
This made GPU cheaper than CPU

Reply:
Thanks for the catch. I think I meant to make it 0.18..